external/java/src/test/java/ch/epfl/dedis/lib/omniledger/OmniledgerRPCTest.java

```diff
@@ -54,7 +55,7 @@ void ping() throws Exception{
 
     @Test
     void updateDarc() throws Exception {
-        SkipBlock previous = ol.getLatest();
+        SkipBlock previous = ol.getSkipchain().getLatestSkipblock();
```

Comment:
It seems like this is going to trigger the new "you shouldn't need this, file an issue" message on every test?